batchUploadResourceCollections.sh

```diff
@@ -1,20 +1,31 @@
 #!/bin/bash
 
-# This bash script is used to batch upload resource collections to the iHub server. It iterates through 
+# This bash script is used to batch upload resource collections to the iHub server. It iterates through  the directories in the src/main/flowResources directory and uploads each resource collection to the server.
 
 # Assumes that you are in the root of the community directory and that it's named like /ihub-community-<COMMUNITY_NAME>
 community=$(basename "$(pwd)" | grep -o '[^-]*$') 
 
 cd ./src/main/flowResources || exit
 
+errorReport=""
 for dir in ./*; do
     if [ -d "$dir" ]; then
         resource=$(basename $dir)
         echo ""
         echo "Uploading collection $resource"
-        uploadResourceCollection.sh "$resource" "$community"
+        response=$(uploadResourceCollection.sh "$resource" "$community")
```

Comment:
`response` will be the output of the `uploadResourceCollection.sh` command.

You could try something like:

```
if ! uploadResourceCollection.sh "$resource" "$community" ; then
   errorReport+="$resource failed to upload.\n"
fi
```

that will use the exit code.

Reply:
Nice, not sure I'd encountered this syntactic nicety before. Thanks again for the tip!

I looked into implementing `try / catch` logic like I'm used to in Python and JS, but it seems it'd require more trouble than it's worth at present.